src/main/scala/edu/berkeley/ce/rockslicing/LoadBalancer.scala

```diff
@@ -0,0 +1,323 @@
+package edu.berkeley.ce.rockslicing
+
+import breeze.linalg
+import breeze.linalg.{DenseVector, DenseMatrix}
+import scala.annotation.tailrec
+
+/** 
+  * Manages initial partitioning of rock volume to maintain load balance among partitions
+  */
+object LoadBalancer {
+
+  /**
+    * Produce processor joints that cut initial rock volume into approximately equal volumes.
+    * These joints will be used to seed the initial Block RDD such that the volume or rock
+    * provided to each process is approximately equal.
+    * @param rockVolume Initial block defining the entire rock volume of interest
+    * @param numSeedJoints Number of joints to process before initiating parallel block cutting.
+    *                      Should be one less than number of cores used.
+    * @return Set of processor joints that will be used to seed the initial RDD
+    */
+  def generateSeedJoints(rockVolume: Block, numSeedJoints: Integer):
```

Comment:
Got rid of the whole seed joint thing across all files. The user now specifies the number of processors that are being used in the analysis and the load balance is done based on that.